saleor/graphql/discount/mutations.py

```diff
@@ -67,6 +69,7 @@ def remove_catalogues_from_node(cls, node, input):
         products = input.get("products", [])
         if products:
             products = cls.get_nodes_or_error(products, "products", Product)
+            products_variant_exist(products)
```

Comment:
Do we need this check here? If we removing connections between products and the node which already exists?